docs/examples/tests/test_using_async_client.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, Dict
```

Comment:
This is the same code that's already in [`docs/examles/testing/get_session_data_async.py`](https://github.com/starlite-api/starlite/blob/main/docs/examples/testing/get_session_data_async.py). What is the purpose of copying this?